### PR TITLE
Add basic Roadmap page

### PR DESCRIPTION
**Pull request by laurakalbag**
A basic Roadmap reflecting our internal Roadmap items. Later on we can make it fancy, but this is the basic content.

Some of them are reworded slightly (based on their issues) for clarity.

Please review considering:
- Is this everything we want to include?
- Should anything here not be included?
- Does grouping by “In Progress”, “Upcoming Q2 2022” and “Later” make sense?

### CODE REVIEW

File: docs/roadmap/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- Bug fixes
+- Discover page
+- Inline implementations
+- Entity snapping
+
+## Upcoming Q2 2022
+
+Features planned for April - June 2022.
+
+### XState
```
Comment (davidkpiano) [inline docs/roadmap/README.md:25]: Should we make this more general? cc. @Andarist 

```suggestion
### Open Source
```

**Review by davidkpiano — APPROVED** [review]
One small suggestion (nit-level), LGTM!

File: docs/roadmap/README.md (hunk 1 above)
Comment (laurakalbag) [reply docs/roadmap/README.md:25]: > Should we make this more general? cc. @Andarist

I was just thinking of using terms that the audience would be familiar with. But if it’s not all XState, then Open Source makes sense.

Comment (davidkpiano) [reply docs/roadmap/README.md:25]: I can go either way - actually, perhaps keeping it as XState makes sense 👍

**Pull request merged**